core/src/main/resources/benchmark-jgroups.xml

```diff
@@ -34,8 +34,6 @@
             <!-- dynamically add/remove protocols from stack -->
             <query value="remove-protocol=MERGE3"/>
             <query value="insert-protocol=MERGE2=above=PING"/>
-            <!-- insert custom protocol (just for illustration -->
-            <query value="insert-protocol=org.radargun.protocols.SLAVE_PARTITION=above=UDP"/>
```

Comment:
Ok, so apparently the SLAVE_PARTITION class was compiling against old jGroups version, I have to override it for ISPN7. Thanks for catching this